### ISSUE TITLE
[Forms & Validation - Make Code - Report Abuse] Form elements must have labels (input[type="url"])

### ISSUE THREAD

**Issue by sonimishra** [microsoft/pxt-microbit#2364]
**User Experience:**
Screen reader user would be affected if screen doesn't associate the label with its control and also this would be create confusion for the screen reader user as they won't be able to get the context of any element or control.

**Repro Steps:**
1. Launch application URL : https://makecode.microbit.org/beta
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Settings" option appearing in header section and select it.
6. Navigate to "Report Abuse" option and select it.
7. Navigate through all the controls appearing under "Report Abuse" dialog.
8. Verify label by using screen reader.

**Actual result:**
Screen reader doesn't associate the label(What is the URL of the offensive project?, Why do you find it offensive?) with it's edit field control on 'Report abuse' pop-up window. It only announces 'URL and edit' which doesn't provide the context of that controls.

**Expected Result:**
Screen reader should associate the label(What is the URL of the offensive project?, Why do you find it offensive?) with it's edit field control on 'Report abuse' pop-up window. It should announce as 'What is the URL of the offensive project edit field and Why do you find it offensive edit field', so that screen reader could get the context of report. 

**MAS Reference:**
[MAS 3.3.2 - Labels or Instructions
](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={4b95a70d-258c-474a-820e-5c8b79d5c0bf})

**Issue:** Form elements must have labels (label: https://dequeuniversity.com/rules/axe/3.2/label?application=msftAI)

**Target application title:** Microsoft MakeCode for micro:bit
Target application url: https://makecode.microbit.org/beta#editor

**Element path:** input[type="url"]

**Snippet:** <input type="url" tabindex="0" placeholder="Enter project URL here...">

**How to fix:**
Fix any of the following:
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Form element does not have an implicit (wrapped) <label>
  Form element does not have an explicit <label>
  Element has no title attribute or the title attribute is empty

**Environment:**
Chrome version 75.0.3770.142
Version 1903 (OS Build 18867.1000)
Platform: Edge, Chrome
Screen Reader: Narrator, JAWS, NVDA

This accessibility issue was found using Accessibility Insights for Web 2.7.1 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights)

![MAS3 3 2_Report Abuse_Label is not properly defined](https://user-images.githubusercontent.com/30456910/62617543-bc951880-b92f-11e9-860e-43fe55a5c48b.jpg)

**Comment by sonimishra** [comment]
Issue has been verified and fixed.
Hence closing the bug.
